src/main/scala/ai/starlake/job/sink/bigquery/BigQueryJobBase.scala

```diff
@@ -400,6 +418,86 @@ trait BigQueryJobBase extends StrictLogging {
     }
   }
 
+  /** Parses the input sql parameter to find all the referenced tables and finds the description of
```

Comment:
We probably need to add a TODO comment stating that Colin description works on BigQuery only and that it should be implemented later on Spark sources